conseil-api/src/main/scala/tech/cryptonomic/conseil/api/ConseilApi.scala

```diff
@@ -183,15 +184,17 @@ class ConseilApi(config: CombinedConfiguration)(implicit system: ActorSystem)
       network <- transformation.overrideNetworks(platform.path, config.platforms.getNetworks(platform.name))
     } yield platform -> network
 
-    cachedDiscoveryOperations.init(visibleNetworks).onComplete {
-      case Failure(exception) => logger.error("Pre-caching metadata failed", exception)
-      case Success(_) => logger.info("Pre-caching successful!")
-    }
+    if (visibleNetworks.nonEmpty) { // At least one blockchain is enabled
```

Comment:
should application run to this point if there isn't enabled network? Maybe we should stop it earlier?

Reply:
That's good questions. I was thinking about that and I decided to let I go, because still you can see the version you are running. I believe you can also take a look at the OpenDoc (shouldn't OpenDoc work in the same way that endpoints are working, meaning show only what is enabled/disabled?), so you can get some value from it, but still no data, no metadata can be served there. Maybe, at some point we would like to host the documentation under separate instance, just to make it a bit more responsive while in the same time, heavy load is on another instances?

I guess it all comes down to whether or not, all of the networks being disabled is just an another state of the application or something what should never happen?

Reply:
Should be done now, please take a look